### PR TITLE
#1406 remove css lint documentation references

### PR DESCRIPTION
**Pull request by jimmyheaddon**
Removed referenced to lint:css, which is currently not supported.

### CODE REVIEW

**Comment by jimmyheaddon** [comment]
As requested in issue #1406

File: docs/general/commands.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ about the bundle size.
 npm run lint
 ```
 
-Lints your JavaScript and CSS.
+Lints your JavaScript.
 
 ### JavaScript
```
Comment (gihrig) [inline docs/general/commands.md]: Looks kind of weird/redundant. How about leaving both CSS and JS sections in the doc for future use, but commenting them out `<!-- ... --> ` until style linting get worked out?

image

Comment (Dattaya) [inline docs/general/commands.md]: ping @jimmyheaddon

**Comment by jimmyheaddon** [comment]
@gihrig how's that look?  I'd avoided comments because they're notoriously tricky in markdown (most MD viewers struggle with the `<!-- -->` syntax), but GitHub handles it, so if that's OK then this should suffice.

If we're not happy that the comments show in other markdown viewers, then I think we're safer deleting the text as per the original commit?

**Comment by gihrig** [comment]
@jimmyheaddon 
>If we're not happy that the comments show in other markdown viewers, then I think we're safer deleting the text as per the original commit?

Agreed, I didn't that, thanks for bringing it up. So just delete the CSS and second JS reference, since `npm run lint` does the same as `npm run lint:js`.

I hadn't noticed before, but this PR needs to be based on `dev` (will be merged into 3.4 before release. You can change this on the PR screen, no need for a new PR).

**Comment by jimmyheaddon** [comment]
Thanks @gihrig, I've changed the base to dev, and have removed the commented out markdown, along with the redundant reference to lint:js.  Hopefully we can get styled-components linting supporting in soon!

**Comment by gihrig** [comment]
@jimmyheaddon Yikes, that branch change brought a whole bunch of unrelated stuff.

Can you open a new PR against current `dev` branch and make these changes please. Sorry for the trouble. 😢

**Comment by jimmyheaddon** [comment]
@gihrig Yeah I didn't think that had gone quite to plan, but I don't have the privileges to correct it inline, so I'll do as you've suggested!

**Pull request closed without merging**